rattan-core/Cargo.toml

```diff
@@ -13,6 +13,7 @@ async-trait = "0.1.68"
 axum = { version = "0.7.5", optional = true }
 backon = "1.2.0"
 bandwidth = "0.3.0"
+bytesize = { version = "1.3.0", features = ["serde"] }
```

Comment:
Do you mean I should modify it like this?
```
bytesize = { version = "1.3.0", optional = true }

serde = [
    "dep:humantime-serde",
    "dep:serde",
    "dep:serde_json",
    "bandwidth/serde",
    "bytesize/serde",
]
```

Reply:
It would be:

```txt
bytesize = { version = "1.3.0"}

serde = [
    "dep:humantime-serde",
    "dep:serde",
    "dep:serde_json",
    "bandwidth/serde",
    "bytesize/serde",
]
```

The deps `bytesize` is used, so it won't be optional. But the `serde` feature of it would be necessary only when our serde feature is enabled.